src/index.html

```diff
@@ -22,6 +22,408 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="header__logo"
+      >
+        <img
+          class="header__logo-moyo"
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__item-button"
+              href="/"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            class="nav__item"
+            data-qa="nav-hover"
+          >
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              href="/"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="#"
+          />
+
+          <div class="card__description card__description--margin--top">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code card__code--margin--top">
+            Product code: 195434
+          </div>
```

Comment:
```suggestion
          <h3 class="card__description card__description--margin--top">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h3>
          <p class="card__code card__code--margin--top">
            Product code: 195434
          </p>
```
Consider leveraging semantic tags